data-reporting-service/organization-service/src/main/java/gov/cdc/etldatapipeline/organization/model/dto/orgdetails/Fax.java

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.etldatapipeline.organization.model.dto.orgdetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgElastic;
+import gov.cdc.etldatapipeline.organization.model.dto.org.OrgReporting;
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString(callSuper = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class Fax implements OrgExtension {
+    private Long faxTlUid;
+    private String faxElpCd;
+    private String faxElpUseCd;
+    private String orgFax;
+
+    public <T> T updateOrg(T org) {
```

Comment:
Just for context, fax specific codes have been implemented in the sproc as they the section is fax specic but for reference- 
fax_elp_cd = 'FAX'
fax_elp_use_cd = 'WP'
These codes should be coming through. 
ELP class_cd = 'TELE' is not part of the json.

Reply:
Updated sproc